module/modmanager/manager.go

```diff
@@ -13,7 +14,7 @@ import (
 	"time"
 
 	grpc_retry "github.com/grpc-ecosystem/go-grpc-middleware/retry"
-	"github.com/pkg/errors"
+	errlib "github.com/pkg/errors"
```

Comment:
flyby: we should prefer using the built-in `errors` package. however, we still rely on a few functions that are only available in this package, so leaving it with a different namespace

Reply:
curious if there's a reason to prefer the built in package, we consciously changed all of the errors packages to the github one a while back

Reply:
ah okay, maybe I'm wrong. it does give us some functions that aren't available in the standard lib, but it looks like it might be based on an old version of go since it's missing some newer functions available in `errors`. In my case, I wanted to use `errors.Join`.

Reply:
based on the name, you may be able to use `multierr.Combine`

Reply:
hmm... the two functions seem really similar. i personally would prefer to use a stdlib version if both are available. but for the sake of minimizing changes in this PR I'll use `multierr.Combine` and we can discuss this as a follow-up

Reply:
Just a note: errors.Join was added in golang 1.20

Reply:
note that https://github.com/pkg/errors has been archived / is in [maintenance mode](https://github.com/pkg/errors?tab=readme-ov-file#roadmap)